src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsworkload/service/staff/JpaBasedSaveEventManagerService.kt

```diff
@@ -54,8 +54,10 @@ class JpaBasedSaveEventManagerService(
   }
 
   private fun auditEventManagerAllocation(allocateCase: AllocateCase, loggedInUser: String, eventManagerEntity: EventManagerEntity) {
-    eventManagerAuditRepository.save(
+    val save = eventManagerAuditRepository.save(
```

Comment:
Does this need to be assigned to a variable?

Reply:
No. Not at all.